test/language/literals/regexp/S7.8.5_A1.1_T1.js

```diff
@@ -12,7 +12,7 @@ description: Without eval
 //CHECK#1
 if (/1/.source !== "1") {
   $ERROR('#1: /1/');
-}   
+}
```

Comment:
are you sure this is not checking for a 2 spaces chars syntax?

Reply:
R-